build.gradle

```diff
@@ -0,0 +1,62 @@
+buildscript {
+    ext {
+        kotlinVersion = '1.1.3-2'
+        springBootVersion = '1.5.4.RELEASE'
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath('se.transmode.gradle:gradle-docker:1.2')
```

Comment:
there are several Gradle plugins for Docker, I like this one: https://github.com/bmuschko/gradle-docker-plugin
it covers more tasks (generate Dockerfile, push to remote registry), and being developed by Gradle employee.